tests/components/technove/test_config_flow.py

```diff
@@ -0,0 +1,32 @@
+"""Tests for the TechnoVE config flow."""
```

Comment:
We should test the config flow more. 100% coverage is required. I am missing the test when an exception is raised and when an entry already exists.

Reply:
Done. I also added two tests for the coordinator.